### DISCUSSION TITLE
acme-device-attest-01 poc

### DISCUSSION THREAD

**Discussion by NickDarvey** [Keyfactor/ejbca-ce#705] [Q&A]
https://docs.keyfactor.com/ejbca/latest/acme-device-attest-01-challenge-poc

I can see this indexed by Google, but I can't find any more information about it or see any references in this repo. Does anyone have any information they can share about this work?

**Comment by primetomas** [comment]
I don't know why this is indexed. ACME device attestation is an upcoming feature. Such a page exist as an internal page, being worked on not fully completed yet. 
ACME is an EJBCA Enterprise feature though, so if you have EJBCA Enterprise you can get information from Keyfactor and probably get a PoC.

**Reply by NickDarvey** [reply]
I'm not an EJBCA Enterprise customer, yet... ACME device attestation is one of the things that'd help me become one!

**Reply by primetomas** [reply]
The feature development is on-going, and PoC testing being documented. With one of the documents being the strangely indexed one :-).

**Reply by alon7786** [reply]
Hi 👋 any news on this feature availability?

**Reply by primetomas** [reply]
We have people working with the IETF standard on board :-). Standard not finished yet, but we have this feature on the roadmap for Q1 in EJBCA Enterprise. PoC is available. The exact date is unknown to me (just cause I'm not on the team).
